client/css/style.css

```diff
@@ -1020,29 +1020,37 @@ button {
 
 #chat .toggle-content img {
 	max-width: 100%;
-	max-height: 250px;
+	max-height: 128px;
 	display: block;
 	margin: 2px 0;
 }
 
 #chat .toggle-content .thumb {
-	max-height: 110px;
-	max-width: 210px;
+	float: left;
+	margin-right: 10px;
+	max-width: 48px;
+	max-height: 36px;
 }
 
 #chat .toggle-content .head {
 	font-weight: bold;
+	white-space: nowrap;
+	text-overflow: ellipsis;
+	overflow: hidden;
 }
 
 #chat .toggle-content .body {
 	color: #999;
 	max-width: 460px;
 	word-break: normal;
 	word-wrap: break-word;
+	overflow: hidden;
+	max-height: 30px;
+	text-overflow: ellipsis;
 }
 
 #chat .toggle-content.show {
-	display: inline-block !important;
+	display: block !important;
```

Comment:
So I tested this and it works just with without the `!important`. However, this rule is redundant with [this Bootstrap one](https://github.com/thelounge/lounge/blob/PR/inline-preview/client/css/bootstrap.css#L1059-L1061) anyway so we should remove it.